apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -438,18 +468,46 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
         {1000,
          [
            Test.create_recovered([{1, 0, 0, alice}], @eth, [{bob, 3}]),
-           Test.create_recovered([{1_000, 0, 0, bob}], @eth, [{alice, 2}])
-         ]},
+           Test.create_recovered([{1_000, 0, 0, bob}], @eth, [{bob, 2}])
+         ]}
+      ])
+
+      assert {[
+                %{"block" => %{"blknum" => 1000}, "txindex" => 1} = tx1,
+                %{"block" => %{"blknum" => 1000}, "txindex" => 0} = tx2
+              ], %{"limit" => 2, "page" => 1}} = transaction_all_with_paging(%{limit: 2})
+
+      blocks_inserter.([
         {2000,
          [
-           Test.create_recovered([{1_000, 1, 0, alice}], @eth, [{bob, 1}])
+           Test.create_recovered([{5, 0, 0, alice}], @eth, [{bob, 10}]),
+           Test.create_recovered([{1_002, 0, 0, bob}], @eth, [{alice, 5}])
          ]}
       ])
 
-      assert [_, _, _] = TestHelper.success?("transaction.all")
+      assert {[tx1, tx2], %{"limit" => 2, "page" => 2}} = transaction_all_with_paging(%{limit: 2, page: 2})
```

Comment:
I don't understand that test. We basically put 4 transactions into the db, and get back four in total in calls for all transactions. What do we really want to test here?

Reply:
Nice catch :+1: